app/controllers/concerns/api/filter_validatable.rb

```diff
@@ -0,0 +1,47 @@
+module API
```

Comment:
maybe we can add a spec for this? I know this is covered in the overall specs thought but worth adding a specific one to test it incase?

Reply:
Well, I thought about it.. but seems like we're testing all API concerns in the features specs instead. I've added a spec anyway. 👍 😄